File: DarkOrbit 10.0/Net/netty/requests/GameplaySettingsRequest.cs
using Ow.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ow.Net.netty.requests
{
    class GameplaySettingsRequest
    {
        public const short ID = 25300;

        public bool DoubleClickAttackEnabled = false;
        public bool AutoChangeAmmo = false;
        public bool AutoStartEnabled = false;
        public bool AutoRefinement = false;
        public bool AutoBoost = false;
        public bool AutoBuyBootyKeys = false;
        public bool QuickSlotStopAttack = false;
        public bool ShowBattlerayNotifications = false;
        public bool varE3N = false;

        public void readCommand(byte[] bytes)
        {
            var parser = new ByteParser(bytes);
            AutoStartEnabled = parser.readBoolean();
            DoubleClickAttackEnabled = parser.readBoolean();
            varE3N = parser.readBoolean();
            AutoBuyBootyKeys = parser.readBoolean();
            parser.readShort();
            AutoBoost = parser.readBoolean();
            QuickSlotStopAttack = parser.readBoolean();
            AutoChangeAmmo = parser.readBoolean();
            ShowBattlerayNotifications = parser.readBoolean();
            AutoRefinement = parser.readBoolean();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/clientCommands/GameplaySettingsRequest.java
package simulator.netty.clientCommands;

import java.io.DataInputStream;
import java.io.IOException;

import simulator.netty.ClientCommand;

public class GameplaySettingsRequest
        extends ClientCommand {

    public static final int     ID                            = 13714;
    public              boolean doubleclickAttackEnabled      = false;
    public              boolean autochangeAmmo                = false;
    public              boolean autoStartEnabled              = false;
    public              boolean autoRefinement                = false;
    public              boolean autoBoost                     = false;
    public              boolean autoBuyBootyKeys              = false;
    public              boolean quickSlotStopAttack           = false;
    public              boolean displayBattlerayNotifications = false;

    public GameplaySettingsRequest(DataInputStream pIn) {
        super(pIn, ID);
    }

    public void read() {
        try {
            this.autoBuyBootyKeys = in.readBoolean();
            this.autoRefinement = in.readBoolean();
            this.autochangeAmmo = in.readBoolean();
            this.quickSlotStopAttack = in.readBoolean();
            this.autoStartEnabled = in.readBoolean();
            this.displayBattlerayNotifications = in.readBoolean();
            this.doubleclickAttackEnabled = in.readBoolean();
            this.autoBoost = in.readBoolean();
        } catch (IOException e) {
        }
    }
}